creed/misc.py

```diff
@@ -0,0 +1,67 @@
+from ark.utils import io_utils
+
+
+def extract_sweep_parameters(sweep_name):
```

Comment:
kind of a future direction here, but pivoting towards using python dataclasses for could help to 'better document' + 'unify usage of' these simple, consistent data structures.  In this case, something like:
```
@dataclass
class VoltageSweepFile:
    voltage: int
    date: str
    time: str


def parse_sweep_name(sweep_name: str) -> VoltageSweepFile:
    ...
    return VoltageSweepFile(voltage, date, time)
```

would allow stuff like:
```
first_sweep = parse_sweep_name(first_fov)
last_sweep = parse_sweep_name(last_fov)

if first_sweep.date != last_sweep.date:
    ...
```

maybe a bit 'over-organizing' in this case, but I think getting into the habit of using these could be good.